### ISSUE TITLE
Include GraphQL operation name in the unhandled request warning

### ISSUE THREAD

**Issue by vldslv-a** [kettanaito/msw#2161]
### Scope

Adds a new behavior

### Compatibility

- [ ] This is a breaking change

### Feature description

Hello! Previously, in unhandled graphql request errors, you could see the operationName of the query, but at the moment this is not possible. Could you provide an example or a function that would fix this?
<img width="874" alt="image" src="https://github.com/mswjs/msw/assets/97732940/9e53fe23-c933-4a62-9ea3-0dcff17d5928">

**Comment by kettanaito** [comment]
Hi, @vldslv-a. Thanks for letting us know.

We've removed the parsing of requests from the `onUnhandledRequest` logic because it's expensive and everybody was paying for it, even if they aren't using GraphQL (there's no way of knowing if a request is a GraphQL request until you attempt to parse it). 

I am a bit hesitant to add the parsing back, even if behind an option (not a fan of configurations in general). 

Would it be more helpful if we instead printed a stringified request body in the message?

```diff
[MSW] Error: intercepted a request without a matching request handler:

  • POST https://app/graphql
    {"query":"query GetUser { ... }","variales":{...}}
```

Reading the request body is not as expensive and will include the GraphQL request information to help you identify the unhandled request. Let me know!

**Comment by vldslv-a** [comment]
> Would it be more helpful if we instead printed a stringified request body in the message?

Yes, that would be nice! It would at least give us more information about unhandled requests, because sometimes they are hard to find.

**Comment by kettanaito** [comment]
@vldslv-a, if you are interested, you can open a pull request with this change! I will help you review the code and merge it for everyone to use.

**Comment by samhoooo** [comment]
Hi @kettanaito @vldslv-a. If no one is working on this, may I make a contribution?

**Comment by bitttttten** [comment]
hey I opened a PR: https://github.com/mswjs/msw/pull/2227
just saw your comment @samhoooo sorry, i jumped on it already! feel free to also contribute and if you have a difference approach i can close my PR (or feel free to make suggestions) 🫶

**Comment by kettanaito** [comment]
## Released: v2.4.0 🎉

This has been released in v2.4.0!

- 📄 [**Release notes**](https://github.com/mswjs/msw/releases/tag/v2.4.0)
- 📦 [npm package](https://www.npmjs.com/package/msw/v/2.4.0)

Make sure to always update to the latest version (`npm i msw@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.
